src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -384,6 +384,14 @@
     <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj">
       <UndefineProperties>%(ProjectReference.UndefineProperties);TargetGroup</UndefineProperties>
     </ProjectReference>
+    <ProjectReference Include="..\..\System.Security.Principal\src\System.Security.Principal.csproj">
```

Comment:
I believe it is because S.R.P is S.R-based rather than CoreLib-based for types, so we can't have references to both CoreLib and the S.R.P package due to type conflicts in S.R. I believe referencing just the runtime library allows this project to use implementations in S.R.P without pulling in the whole set of types. At least that's my limited understanding.